types/puppeteer/index.d.ts

```diff
@@ -1272,6 +1272,11 @@ export interface PageEventObj {
    * Puppeteer can respond to the dialog via Dialog's accept or dismiss methods.
    */
   dialog: Dialog;
+  /**
+   * Emitted when the initial HTML document has been completely loaded and parsed, 
+   * without waiting for stylesheets, images, and subframes to finish loading.
+   */
+  domcontentloaded: undefined;
```

Comment:
I think `never` or `void` might be better here actually.

Reply:
Agreed! `void` will be better, will replace this.